build/index.js

```diff
@@ -58,8 +66,12 @@ exports.onCreateProject = function (api, app, config, cb) {
         currStrDom.val = googleConf[attrName];
       }
     }
-
-    return fs.outputFileAsync(path.join(androidProjectPath, 'res/values', strings_file), xmlStr.toString(), 'utf-8');
+    return fs.outputFileAsync(path.join(androidProjectPath,'../../',
```

Comment:
all other modules we are using this variable name as outeutpath. can we change here also for easy understanding.

Reply:
Changed back